PSKoans/Data/Advice/OOP/OOPIntroduction.Advice.json

```diff
@@ -0,0 +1,33 @@
+{
+    "Title": "Object-Orientated Programming (OOP) Introduction",
+    "Content": [
+
+        "PowerShell is object-orientated, but what does this mean? Everything is an object! ",
+        "\n",
+        "Objects are a way for us to build programs to scale, and OOP is standard in software development. For the purposes of this advise file there's only three fundamentals you need to know:",
+        "\n",
+        "\tObjects are blueprints",
+        "\tObjects have properties",
+        "\tObjects have methods",
+        "\n",
+        "If you've already done the Get-Member Koan then you'll know we can use this cmdlet to discover what blueprint an item is using, what properties it has and what methods it may execute. For example we may take any string and pass it through into Get-Member in order to get the below result:",
```

Comment:
```suggestion
        "If you've already completed the Get-Member topic, then you'll know we can use this cmdlet to discover what blueprint an object is using, what properties it has, and what methods it may execute. For example, we may take a string and pass it into Get-Member in order to get the below result:",
```

Reply:
🦄